README.md

```diff
@@ -1 +1,57 @@
-# capstone-project
\ No newline at end of file
+# Global CTF Competition Summit
+
+ This is a website that publish the ctf competition summit for the global Community
+ ![CTF Summit (1)](https://user-images.githubusercontent.com/26603127/164704299-9f998c68-1f49-4969-976c-3da7b1216bf7.png)
+
+
+ ### Built With
+
+ - Html,
+ - css
+ - Javascript
+
+ ### Live Demo
+ [freddykaberuka.github.io](https://freddykaberuka.github.io/capstone-project/)
+
+## Getting Started
+
+### Setup
+
+Clone or Download Repository
+Extract the download Folder
+
+### Install
+
+after cloning Repository open the folder and run npm install to install all the modules in your local storage
+
+### Usage
+
+Open the extracted folder and double click on index.html. BOOM!!
+
+and for the contributer i recommended to use [VSCode](https://code.visualstudio.com/) as text editor and Live Server installed on it.
+
+## 👤 **Freddy KABERUKA**
+
+- GitHub: [@freddykaberuka](https://github.com/freddykaberuka)
+- Twitter: [@trojanx07](https://twitter.com/trojanx07)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kaberuka-freddy-853b08153/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ Optional ] We are sorry for not pointing it earlier, but you should give credit to [Cindy Shin in Behance](https://www.behance.net/adagio07) for the original design idea as described in the project requirement. 
 
  > The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.